src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -89,7 +91,8 @@ const LinkAndSocialMedia: React.FC<LinkProps> = ({username}) => {
             <Text
               style={{
                 fontSize: 12,
-                color: COLORS.white,
+                fontFamily: fonts.inter[400],
+                color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fontFamily and color properties have been updated. Make sure that these changes are consistent with the design system and improve readability.
```diff
-                color: COLORS.white,
+                fontFamily: fonts.inter[400],
+                color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->